edx-theme migration and fixes from cypress to dogwood version

ms-hukoyun opened the pull request:
Microsoft's Stanford edx-theme created for cypress version was not working properly on dogwood. Issues were:
1) In any course wiki page was broken
2) In user menu there was no dashboard menu, Profile and Account Settings was not working
3) After signing-on the right-top button for user name and user menu didn't look good CSS was totally broken. 
4) There were inconsistencies because we were overwriting html files like navigation.html, register.html, index.html, main.html but we used cypress version of these files and in dogwood there are changes in these files and we were overwriting them.

Using dogwood version of these html files I reapplied the edX-theme changes we did for cypress . I fixed the CSS issues. I also made changes for Country or Region in register page. 

Now Stanford edX-theme is totally fixed and functional for dogwood version and ready for production.

@Microsoft/lex

ms-hukoyun committed:
Changes needed for Microsoft Stanford theming to work properly on dogwood version of Open edX.

ms-hukoyun committed:
We are not changing anything in this file. So no need to override this file. Removing from theme package

ms-hukoyun committed:
Fixed some CSS issues on Sign On buton and dropdown menu

ms-hukoyun committed:
Fixed the hover issue on signed on menu button

btelnes commented on templates/theme-footer.html:
```diff
@@ -10,27 +10,21 @@
-        </ol>
-      </nav>
+        <li><a href="https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/en-us.aspx">Trademarks</a></li> 
+        </ol>      </nav>
```
Nit: Alignment of  </nav>

btelnes commented on static/sass/_default.scss:
```diff
@@ -348,7 +348,7 @@ div.main-cta > a.register:hover {
 // The user dropdown menu on the right hand side
 header.global .user a.user-link{
-    padding:5px 12px 10px 10px;
+    padding:5px 12px 10px 10px !important;
```
Nit: Add space after colon

btelnes commented on templates/main.html:
```diff
@@ -48,7 +47,7 @@
   % if not allow_iframing:
       <script type="text/javascript">
-        function setCookie(cname, cvalue, exdays) {
+	function setCookie(cname, cvalue, exdays) {
```
Nit: indent

chenriksson replied:
Should use spaces for indents, not tabs... for consistency.

btelnes commented on templates/main.html:
```diff
@@ -64,7 +63,7 @@
              return "";
         }
-        function closeCookieBanner() {
+	function closeCookieBanner() {
```
Nit: indent

ms-hukoyun committed:
Fixed some space and indentation issues.

ms-hukoyun commented:
Brian thanks for the feedback. I fixed the spacing and indentation issues and commited/pushed changes.

chenriksson commented on templates/register.html:
```diff
@@ -259,7 +259,7 @@ <h2 class="sr">${_("Additional Personal Information")}</h2>
           <li class="field-group">
           <div class="field ${settings.REGISTRATION_EXTRA_FIELDS['country']} select" id="field-country">
-              <label for="country">${_("Country")}</label>
+              <label for="country">${_("Country or Region")}</label>
```
Looks like index.html, main.html and navigation.html were updated for Dogwood. Can you also look if register.html should be updated - looks like it's missing some changes.

chenriksson commented on templates/theme-header.html:
```diff
@@ -5,5 +5,6 @@
 ## TODO: this will eventually be moved to the Sass
 <%block name="navigation_logo">
-    <img src="${static.url('themes/default/images/ms-logo.png')}" width="108" height="23" vspace="15" alt="Home Page" /> 
+	<img src="${static.url('themes/default/images/ms-logo.png')}" width="108" height="23" vspace="15" alt="Home Page" />
```
Would revert. We should continue using spaces instead of tabs for consistency.

ms-hukoyun committed:
Replaced tabs with spaces for consistency. Retreived the register.html and made the change again.

ms-hukoyun commented:
Christy thanks for the feedback. I replaced the tabs with spaces in theme-header.html and main.html. 
I got the register.html and changed label "Country" to "Country or Region" again and this is the only change we made in this file. It looks like it is same and nothing was missing in my previous check in. Just to be sure I used the latest register.html from oxa/master branch of edx-platform repo.
